spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -74,6 +75,29 @@ object Engine {
     _conf
   }
 
+  def waitAllExecutorsStartedInMesos: Unit = {
+    if (System.getProperty("spark.master").toLowerCase.startsWith("mesos")) {
+      val sc = SparkContext.getOrCreate(new SparkConf()
```

Comment:
Do you test it? I don't think it work because there will be two spark context in the JVM